src/Mod/PartDesign/Gui/TaskPipeParameters.cpp

```diff
@@ -414,7 +414,7 @@ bool TaskPipeParameters::referenceSelected(const SelectionChanges& msg) const {
                     return false;
             }
 
-            static_cast<PartDesign::Pipe*>(vp->getObject())->Spine.setValue(
+            static_cast<PartDesign::Pipe*>(vp->getObject())->Spine.setValue(


```

Comment:
The linebreak has a strange character

Reply:
It's the carriage return character. Probably a bug in emacs adding windows line ending where it's not needed.